frontend/micro-ui/web/micro-ui-internals/packages/components-core/src/atoms/HeaderBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const HeaderBar = (props) => {
+  return (
+    <div className="header-wrap" style={props?.style ? props.style : {}}>
```

Comment:
what is App bar and this header bar

Reply:
@jagankumar-egov replaced with header component